prow/github/types.go

```diff
@@ -62,9 +62,30 @@ type User struct {
 	Email string `json:"email"`
 }
 
+type EventAction string
```

Comment:
Yes absolutely -- is this a change we want? I am really unclear on why this part of the project does not use `go-github` and why a lot of these were stringly-typed in the first place. If I'm missing a design consideration for `prow` I can also just ditch this commit.